### PR TITLE
WIP: Adding recipe for SV2

### PR DESCRIPTION
**Pull request by holtgrewe**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/sv2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+source:
+  fn: sv2-{{ version }}.tar.gz
+  url: https://github.com/dantaki/SV2/archive/sv2v1.4.1.tar.gz
+  sha256: "{{ sha256sum }}"
+  patches:
+  - sv2.patch
+
+build:
+  skip: True # [py3]
```
Comment (druvus) [inline recipes/sv2/meta.yaml]: This should be py3k or not py27

File: recipes/sv2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{% set version = "1.4.1" %}
+{% set sha256sum = "16fdc9e59f837549f149906fe4e771da40265e52d165460ecf6ac8949df7397b" %}
+
+package:
+  name: sv2
+  version: {{ version }}
+
+source:
+  fn: sv2-{{ version }}.tar.gz
+  url: https://github.com/dantaki/SV2/archive/sv2v1.4.1.tar.gz
```
Comment (druvus) [inline recipes/sv2/meta.yaml]: You could use {{ version }} also in the url

File: recipes/sv2/meta.yaml (hunk 1 above)
Comment (holtgrewe) [reply recipes/sv2/meta.yaml]: So the line should read `skip: True # [py3k]`?

File: recipes/sv2/meta.yaml (hunk 2 above)
Comment (holtgrewe) [reply recipes/sv2/meta.yaml]: Good catch!

File: recipes/sv2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  run:
+  - sv2 --help
+  - sv2train --help
+
+about:
+  home: https://github.com/dantaki/SV2
+  summary: Support Vector Structural Variation Genotyper
+  license: MIT
```
Comment (druvus) [inline recipes/sv2/meta.yaml]: please add `license_file: LICENSE`

File: recipes/sv2/meta.yaml (hunk 1 above)
Comment (druvus) [reply recipes/sv2/meta.yaml]: yes

File: recipes/sv2/meta.yaml (hunk 3 above)
Comment (holtgrewe) [reply recipes/sv2/meta.yaml]: Thanks!

Comment (holtgrewe) [reply recipes/sv2/meta.yaml]: By the way, is it possible to somehow access the conda package created by CI?

**Comment by holtgrewe** [comment]
@druvus What do you think? Does this look good to you?

**Comment by holtgrewe** [comment]
Hm, the built package is too slow, I think this needs more work.

**Comment by bgruening** [comment]
> Hm, the built package is too slow, I think this needs more work.

What do you mean with slow?

**Comment by druvus** [comment]
@holtgrewe is this getting ready to be merged?

**Comment by holtgrewe** [comment]
Hi. I need support from upstream... Some files are missing apparently

**Pull request closed without merging**